source/backend/api/Areas/Projects/Controllers/ProjectController.cs

```diff
@@ -110,14 +106,14 @@ public IActionResult AddProject(ProjectModel projectModel)
         [Produces("application/json")]
         [ProducesResponseType(typeof(ProjectModel), 200)]
         [SwaggerOperation(Tags = new[] { "project" })]
-        public IActionResult UpdateProject([FromRoute]long id, [FromBody] ProjectModel model)
+        public IActionResult UpdateProject([FromRoute] long id, [FromBody] ProjectModel model)
         {
             if (id != model.Id)
             {
                 return BadRequest();
             }
 
-            var updatedProject = _projectService.Update(id, _mapper.Map<Dal.Entities.PimsProject>(model));
+            var updatedProject = _projectService.Update(_mapper.Map<Dal.Entities.PimsProject>(model));
```

Comment:
@eddherrera thought you changed this. Let me know if you think this needs to be changed back given id parameter not used

Reply:
It's all right.